operator_toolkit/operation/utils.py

```diff
@@ -74,22 +79,23 @@ class HttpMethods(Enum):
 class FileManager:
     
     @staticmethod
-    def load_config(force_local=False):
-        """Load the operational sites of Axiom. First search for an s3 config file and if
-        the file is not found checks for local config files.
+    def load_config():
+        """Load the operational sites of Axiom. First search for a local config file and if
+        the file is not found checks for s3 config files.
 
         :return: List of operational sites
         :rtype: [dict, None]
         """
-        # load s3 config with operational sites
-        config = FileManager.load_file(OPERATIONAL_SITES_S3_PATH)
-        if config and not force_local:
-            return config
-
         # load local config with operational sites if not s3
         config = FileManager.load_file(OPERATIONAL_SITES_LOCAL_PATH)
         if config:
             return config
+
+        # load s3 config with operational sites
+        config = FileManager.load_file(OPERATIONAL_SITES_S3_PATH)
+        if config:
+            return config
+
```

Comment:
So the result of having this `load_file()` method be nullable (can return something falsy) is that you're able to play this trick where you exit early if you find the local file and only attempt the S3 load if you make it there. This works, but you've violated DRY, it's almost the exact same snippet.

Ideally, we should make one assignment and one conditional return. We can leverage the short-circuit nature of boolean expressions (lazy evaluation) to achieve the same thing as the early exit.
```suggestion
        # load config with operational sites
        config = (FileManager.load_file(OPERATIONAL_SITES_LOCAL_PATH) or
                  FileManager.load_file(OPERATIONAL_SITES_S3_PATH))
        if config:
            ...  # deal with failed config loading
        return config

```

The other thing to note is that I typically prefer to have the "happy path" return be the default and use conditionals to do error handling.

Reply:
mmm nice, I like it, thanks 👌